paasta_tools/list_tron_namespaces.py

```diff
@@ -34,14 +34,24 @@ def parse_args():
         default=tron_tools.DEFAULT_SOA_DIR,
         help="Use a different soa config directory",
     )
+    parser.add_argument(
+        "-e",
+        "--executors",
+        default=list(tron_tools.EXECUTOR_TYPE_TO_NAMESPACE.keys()),
```

Comment:
Yes, `list_tron_namespace` would have returned those that only had `executor: ssh` that may not be spark jobs previously, so we shouldn't disturb that status quo imho.

It might be true this argument has no practical effect (i.e. there are no real `executor: ssh` jobs right now that *aren't* `spark-run` commands), but unless we also enforce that (no ssh jobs except spark jobs) going forward we should probably not exclude them from being listed by the `list_tron_namespaces` command.